README.md

```diff
@@ -73,6 +73,21 @@ The project is organized into the following directories and files:
 4. Rename `.env.example` to `.env` and fill in the values
 5. Run `make server` and `make worker` to start the web server and the Celery worker
 
+
+## Pre-Commit Setup
+
+If you are using this project then pre-commit setup would be very helpful for checking your codebase. In short, pre-commit is a tool that allows developers to define and apply automated checks to their code before they commit it to a version control system. You can find more info [here](https://pre-commit.com)
+
+
+```commandline
+pip install pre-commit
+pre-commit --version
```

Comment:
Can you add `pre-commit` dependency to `requirements/dev.txt`? All the dev-only dependencies are listed in that file, and there is a make command that conveniently installs them for local development (`make install_dev`).

You can remove these two instructions from the Readme as they won't be necessary.

Reply:
Yes rightly said. Changed!